README.md

```diff
@@ -1,5 +1,11 @@
-Instagram Challenge
+[![Build Status](https://travis-ci.org/joejknowles/instagram-challenge.svg?branch=master)](https://travis-ci.org/joejknowles/instagram-challenge)
+~~Instagram~~ Quitefastgram Challenge
 ===================
+####Find it on heroku
+
+http://quitefastgram.herokuapp.com/
+
```

Comment:
In the README, add a section for downloading and setting up the app to run locally, and another section for setting up the app to run tests.

# Getting started
A live version is available [here](http://quitefastgram.herokuapp.com). To run in development:
```bash
git clone <repository> # get the local version
bundle install # install dependencies
rake db:migrate # create database tables
```

To run the app:

```bash
rails s
```

# Testing
The app has full test coverage. To run tests:
```bash
rspec spec
```